modules/publicize/publicize-jetpack.php

```diff
@@ -442,6 +444,25 @@ function save_publicized( $post_ID, $post, $update ) {
 		}
 	}
 
+	function set_post_flags( $flags, $post ) {
+		$flags[ 'publicize_post' ] = false;
```

Comment:
> When referring to array items, only include a space around the index if it is a variable

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/

I'd recommend installing Code Sniffer rules on your machine to avoid such issues in the future:
https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards#installation